rest/resource-server/src/docs/asciidoc/licenses.adoc

```diff
@@ -97,4 +97,15 @@ A `DELETE` request will delete all licenses info.
 include::{snippets}/should_document_delete_all_license_info/curl-request.adoc[]
 
 ===== Example response
-include::{snippets}/should_document_delete_all_license_info/http-response.adoc[]
\ No newline at end of file
+include::{snippets}/should_document_delete_all_license_info/http-response.adoc[]
+
+[[import-SPDX-info]]
+==== Import SPDX information
+
+A `IMPORT` request will import SPDX info.
+
```

Comment:
Change message to --> A `POST` request will import SPDX info.

Reply:
done.